src/ducks/SANCharts/components/newSignalLine/SignalLine.module.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@santiment-network/ui/mixins';
+@import '~@santiment-network/ui/variables';
+
+.line {
+  border: 1px dashed var(--rhino);
+}
+
+.tooltip {
+  padding: 2px 8px;
+  background: var(--rhino);
+  color: var(--casper);
+  position: relative;
+  left: 8px;
+  bottom: 5px;
+  border-radius: $border-radius;
+  display: table;
```

Comment:
`display: table` why?)

Reply:
Container has full width, but child needs to fill only available content with text. Display:table solve this problem. flex/block/etc filled 100% of container.